src/npm/hook.js

```diff
@@ -131,20 +129,13 @@ const send = (data, port) => {
   )
 }
 
-const getRequiredMeasures = (components, measures) => {
-  let requiredMeasures = []
-
-  if (!Array.isArray(components)) {
```

Comment:
There is no need for this, since in L73, we already were doing this checkl

Reply:
Oops! Ya totally 👍